Error while checking build/O2/o2 for 44be314ebb1bcd2130ccd77f5344d305cd6c3bcc at 2022-10-10 23:26:
```
## sw/BUILD/O2-latest/log
100% tests passed, 0 tests failed out of 457
100% tests passed, 0 tests failed out of 102


## sw/BUILD/QualityControl-latest/log
27/38 Test #33: multinode_test ..........................***Failed   35.76 sec
[3861:qc-task-TST-MultiNodeRemoteTest51392]: 2022-10-10 21:25:55.919577 !!! [1;31mError - Could not find the DPL InfoLogger[0m
[3863:qc-check-TST-MultiNodeRemoteTest]: 2022-10-10 21:25:55.919340 !!! [1;31mError - Could not find the DPL InfoLogger.[0m
[3862:qc-check-TST-MultiNodeLocalTest]: 2022-10-10 21:25:55.927935 !!! [1;31mError - Could not find the DPL InfoLogger.[0m
[3873:qc-task-TST-MultiNodeLocalTest36090]: 2022-10-10 21:25:55.999518 !!! [1;31mError - Could not find the DPL InfoLogger[0m
[ERROR] websocket_server_callback: Error while reading from websocketconnection reset by peer
[ERROR] SEVERE: Device internal-dpl-clock (3857) returned with 128
97% tests passed, 1 tests failed out of 38
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/10044/44be314ebb1bcd2130ccd77f5344d305cd6c3bcc/build_O2_o2/pretty.html).

--- a/Framework/Foundation/src/RuntimeError.cxx
+++ b/Framework/Foundation/src/RuntimeError.cxx
@@ -158,12 +158,12 @@ void demangled_backtrace_symbols(void** stackTrace, unsigned int stackDepth, int
         char path[1024];
 
         fp = popen(syscom, "r");
-        if (fp == NULL) {
+        if (fp == nullptr) {
           dprintf(fd, "-- no source could be retrieved --\n");
           continue;
         }
 
-        while (fgets(path, sizeof(path) - 1, fp) != NULL) {
+        while (fgets(path, sizeof(path) - 1, fp) != nullptr) {
           dprintf(fd, "    %s", path);
         }
 

